stable/jenkins/templates/jenkins-master-ingress.yaml

```diff
@@ -1,5 +1,9 @@
 {{- if .Values.master.ingress.enabled }}
-apiVersion: {{ .Values.master.ingress.apiVersion }}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
```

Comment:
Could you set this to {{ .Values.master.ingress.apiVersion }} so that the chart keeps compatibility?

Reply:
@torstenwalter I have made the changes suggested by you.